docs/src/equations.md

```diff
@@ -144,21 +147,21 @@ is discretized using
 \frac{\partial}{\partial t} \rho e \approx
 - D_h[ (\rho e + p) (\boldsymbol{u}_h + I^c(\boldsymbol{u}_v))]
 - D^c_v \left[
-  I^f(\rho) U^f\left(\boldsymbol{u}_v, \frac{\rho e + p}{\rho} \right)
-  + (\rho e + p) I^f(\boldsymbol{u}_h)
+  WI^f(J, \rho) U^f\left(\boldsymbol{u}_v, \frac{\rho e + p}{\rho} \right)
+  + WI^f(J, (\rho e + p) \boldsymbol{u}_h)
```

Comment:
I'm concerned this is not correct. The mass weighted averaging should just apply to the advective velocities. (We need to translate Taylor et al.'s Eq. (48) into our setting. I believe it needs to be
```suggestion
  +   WI^f(J, \rho \boldsymbol{u}_h) I^f(e + p/\rho)
```
(This may have led to the diffusive behavior we saw. @Zhengyu-Huang please check this.)

Reply:
How was this working before without interpolation for `(\rho e + p)` in the last term, given that `(\rho e + p)` lives on centers but we need a face quantity here?